docs/operation/volume_workflow/volume_creation_deletion_gui.rst

```diff
@@ -0,0 +1,62 @@
+Volume Workflow GUI Operations
+==============================
+This section highlights the creation and deletion of MetalK8s volume objects
+using the MetalK8s GUI.
+
+Volume Creation GUI
+-------------------
+
+.. important::
```

Comment:
Same remark as for CLI: this is not required, only useful if people want to create their own so can be mentioned before (as I suggested for CLI)